Documents/Help/Changelog.md

```diff
@@ -1,6 +1,7 @@
 # <img src="https://github.com/Krypton-Suite/Standard-Toolkit/blob/master/Krypton.png"> Standard Toolkit - Changelog
 
 ## 2101-09-0x - Build 2109 - September 2021 (Canary)
+* New `KryptonBorderedLabel`, moved control from the extended toolkit.
```

Comment:
Nasty !

Reply:
The possible other way to deal with this is to hide it from the toolbox in the extended toolkit

Reply:
Or not put it into standard ..